packages/services/src/index.ts

```diff
@@ -61,7 +83,7 @@ export class Services implements ServicesInterface {
   }
 
   private async initializeWalletServices(): Promise<WalletServices> {
-    const { walletId, fullViewingKey } = await this.config.getWallet();
+    const { walletId, fullViewingKey, idbVersion: dbVersion } = await this.config;
     const params = await this.querier.app.appParams();
     if (!params.sctParams?.epochDuration) throw new Error('Epoch duration unknown');
```

Comment:
pairing with @jessepinho i think it's not appropriate to get epoch duration here but that should be a separate issue